### PR TITLE
Profile Ui graph and piechart fixed

### PR DESCRIPTION
**Pull request by rudraneel18**

### CODE REVIEW

File: app/src/main/java/com/example/codechefeventsapp/activities/MainActivity.java [hunk 1]
```diff
@@ -38,11 +38,11 @@ private void setupBottomNavBar() {
     }
 
     private void setupToolbar() {
-        setSupportActionBar(toolbar);
+       // setSupportActionBar(toolbar);
```
Comment (gargVader) [inline app/src/main/java/com/example/codechefeventsapp/activities/MainActivity.java]: Why comment this out?
Was the app not building because of this?

**Comment by rudraneel18** [comment]
yes on my device it was not building so i commented out those

**Comment by gargVader** [comment]
Okay. You can comment it out on your local machine. For the purpose of the main repo, just uncomment it and make a commit. Line 41 and 45

**Comment by rudraneel18** [comment]
So that will be another pr right?

**Comment by gargVader** [comment]
No. Just update your repo. Changes will be reflected here

**Comment by rudraneel18** [comment]
Done, could you please confirm it.

**Comment by gargVader** [comment]
Well done. Thanks!

**Pull request merged**